opentelemetry-resource-detector/Cargo.toml

```
@@ -0,0 +1,18 @@
+[package]
+name = "opentelemetry-resource-detector"
```

```suggestion
name = "opentelemetry-resource-detectors"
```

Would it make sense to use plural name, given this has multiple detectors!

---

Make sense to me, changed in https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/49/commits/380ca691c838d27667c03df30b5d68f0d8b72596